backend/src/main/java/se/chalmers/dat265/group1/storage/FileUtil.java

```diff
@@ -0,0 +1,12 @@
+package se.chalmers.dat265.group1.storage;
+
+import java.io.*;
+
+public class FileUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)